### DISCUSSION TITLE
Should psn-api have a Discord server?

### DISCUSSION THREAD

**Discussion by wescopeland** [achievements-app/psn-api#125] [General]
Something I am wondering now that the lib is gaining traction. This would likely be quite small, but might be helpful for quick exchange of support info and maintainer communication.

A potential downside is Discord communication isn't easily searchable and tends to be quite ephemeral.

**Comment by TheYuriG** [comment]
I can support discussions/questions in a Discord server if this goes forward, but I'm fairly mild in either direction.
